src/drivers/magnetometer/lis2mdl/lis2mdl_main.cpp

```diff
@@ -84,7 +84,7 @@ void LIS2MDL::print_usage()
 	PRINT_MODULE_USAGE_SUBCATEGORY("magnetometer");
 	PRINT_MODULE_USAGE_COMMAND("start");
 	PRINT_MODULE_USAGE_PARAMS_I2C_SPI_DRIVER(true, true);
-	PRINT_MODULE_USAGE_PARAMS_I2C_ADDRESS(0x1e);
+	PRINT_MODULE_USAGE_PARAMS_I2C_ADDRESS(LIS2MDLL_ADDRESS);
```

Comment:
Module documentation doesn't support macros: https://github.com/PX4/PX4-Autopilot/actions/runs/3348518838/jobs/5547671389

Reply:
Thanks, it's fixed